apps/nextra/pages/en/build/cli/trying-things-on-chain/ledger.mdx

```diff
@@ -117,3 +119,499 @@ yes
 After you have approved publishing this package you will be prompted to sign the transaction on your Ledger device. Once signed, the package will be published to the network!
 
 One error you might run into is `Error: Wrong raw transaction length`. This means that the transaction or package size was too big for your device to sign. Currently the Aptos Ledger app can only support transactions that are smaller than 20kb. The `Ledger Nano S` device has less memory than that, which is why it is more likely to produce this error.
+
+## Authentication key rotation
+
+If you have an active account that is not secured using a hardware wallet, then
+you may wish to rotate the account's authentication key so that it corresponds
+to a [BIP44 account index] private key held on your Ledger.
+
+Alternatively, if you have an account linked with a Ledger hardware wallet that
+you wish to publish a large package from, you might want to temporarily rotate
+the account's authentication key to a hot key to avoid memory issues.
+
+This tutorial will walk you through both scenarios.
+
+<Callout type="warning" emoji="❗">
+Before you start this tutorial make sure you have completed the
+[key rotation guide](../../advanced-guides/key-rotation.mdx).
```

Comment:
All of the relative links to the key rotation guide do not work for me when testing it in the preview.

Reply:
It looks like `advanced-guides` was renamed to `guides`

Resolved in c32f593